ci-operator/jobs/sclorg/ansible-tests/sclorg-ansible-tests-master-periodics.yaml

```diff
@@ -777,21 +692,10 @@ periodics:
   labels:
     ci-operator.openshift.io/cloud: ibmcloud
     ci-operator.openshift.io/cloud-cluster-profile: ibmcloud-cspi-qe
-    ci-operator.openshift.io/variant: dotnet-ocp4.17-lp-interop
+    ci-operator.openshift.io/variant: dotnet-ocp4.18-lp-interop
     ci.openshift.io/generator: prowgen
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
-  name: periodic-ci-sclorg-ansible-tests-master-dotnet-ocp4.17-lp-interop-dotnet-interop-ibmcloud
-  reporter_config:
-    slack:
-      channel: '#team-dotnet-eng'
-      job_states_to_report:
-      - success
-      - failure
-      - error
-      report_template: '{{if eq .Status.State "success"}} :slack-green: Job *{{.Spec.Job}}*
-        ended with *{{.Status.State}}*. <{{.Status.URL}}|View logs> {{else}} :failed:
-        Job *{{.Spec.Job}}* ended with *{{.Status.State}}*. <{{.Status.URL}}|View
-        logs> {{end}}'
```

Comment:
Looks like the slack config was removed for 4.18. You may need to re-add after running the pj-rehearse